Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM kbase/kbase:sdkbase2.latest
-MAINTAINER KBase Developer [Dylan Chivian (DCChivian@lbl.gov)]
+FROM kbase/sdkbase2:python
+MAINTAINER KBase Team
 # -----------------------------------------
 
 # RUN apt-get update
 RUN pip install --upgrade pip
 
 # Install MEGAHIT
 WORKDIR /kb/module
-RUN \
-  git clone https://github.com/voutcn/megahit.git && \
-  cd megahit && \
-  git checkout tags/v1.1.1 && \
-  make
+ARG filename=MEGAHIT-1.2.8-Linux-x86_64-static
+RUN cd /opt && \
+  curl -L -O https://github.com/voutcn/megahit/releases/download/v1.2.8/${filename}.tar.gz && \
```

Comment:
The latest version is 1.2.9, it has some fixes, you should update this and update the changelog and display.yaml

Reply:
Good catch, that must be recent. I'll update once I have a few mins

Reply:
Updated